### PR TITLE
#26, test GHC 8.4

### PR DESCRIPTION
**Pull request by ndmitchell**
Done blind, but from local experiments.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -44,6 +44,13 @@ jobs:
       - bash: stack test || stack test || stack test
         # ghcide stack tests are flaky, see https://github.com/digital-asset/daml/issues/2606.
         displayName: 'stack test'
+      - bash: stack setup --stack-yaml=stack84.yaml
```
Comment (cocreature) [inline azure-pipelines.yml]: This isn’t quite right. This should be a separate CI job so we can run them in parallel.

Comment (ndmitchell) [reply azure-pipelines.yml]: I don't have the skills to do that, at the very least without active experimentation, so if you want that before merge I suggest we close this PR.

Comment (cocreature) [reply azure-pipelines.yml]: If you don’t mind, I can push to your PR and try to fix it when I find the time.

Comment (ndmitchell) [reply azure-pipelines.yml]: Happily!

**Pull request merged**